packages/codemods/transforms/v4/icon/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+module.exports = function ({ source /*, path*/ }, { parse, visit }) {
+  const ast = parse(source);
+
+  const updateIsInlineBlockAttribute = (attributes) => {
+    const indexOfIsInlineBlockAttr = attributes.findIndex((a) => a.name === '@isInlineBlock');
+    const hasIsInlineBlockAttr = indexOfIsInlineBlockAttr !== -1;
```

Comment:
This is used only once, you can test directly `indexOfIsInlineBlockAttr !== -1` in the `if(...)` and remove this definition